.cirrus.yml

```diff
@@ -165,6 +165,27 @@ task:
   test_all_script:
     - ./dev/bots/deploy_gallery.sh
 
+task:
+  use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true'
+  name: add2app-macos
+  depends_on:
+    - analyze
+  env:
+    # Name the SDK directory to include a space so that we constantly
+    # test path names with spaces in them.
+    CIRRUS_WORKING_DIR: "/tmp/flutter add2app"
+    SHARD: add2app_test
+    COCOAPODS_DISABLE_STATS: true
+  osx_instance:
+    # note: the high sierra image doesn't seem to include a simulator.
+    image: mojave-xcode-10.1
```

Comment:
Does it work to just have this `osx_instance` tag inside of one of the matrix nodes below?  I assume not, but you never know..  That way you could make sure this is being executed with most of the other stuff set up the same way (like artifact/package/pub caching, etc.).

Reply:
I've bumped all our macOS to mojave and we'll see if it works.  Unfortunately right now this is blocked on a issue with analysis for the analyzer package.